Editor/src/Program.bf

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace SteelEditor
+{
+	class Program
+	{
+		public static int Main(String[] args)
+		{
+			Console.WriteLine("Hello Steel Editor user!");

```

Comment:
This is actually not necessary because I am making some basic GUI for the editor application. Just make a sandbox workspace somewhere else and add the steel projects.